htdocs/core/lib/date.lib.php

```diff
@@ -658,20 +658,21 @@ function getGMTEasterDatetime($year)
 }
 
 /**
- *	Return the number of non working days including saturday and sunday (or not) between 2 dates in timestamp.
+ *  Return the number of non working days including Friday, Saturday and Sunday (or not) between 2 dates in timestamp.
  *  Dates must be UTC with hour, min, sec to 0.
- *	Called by function num_open_day()
+ *  Called by function num_open_day()
  *
- *	@param	    int			$timestampStart     Timestamp start (UTC with hour, min, sec = 0)
- *	@param	    int			$timestampEnd       Timestamp end (UTC with hour, min, sec = 0)
- *  @param      string		$country_code       Country code
- *	@param      int			$lastday            Last day is included, 0: no, 1:yes
- *  @param		int			$includesaturday	Include saturday as non working day (-1=use setup, 0=no, 1=yes)
- *  @param		int			$includesunday		Include sunday as non working day (-1=use setup, 0=no, 1=yes)
- *	@return   	int|string						Number of non working days or error message string if error
+ *  @param	int			$timestampStart		Timestamp start (UTC with hour, min, sec = 0)
+ *  @param	int			$timestampEnd		Timestamp end (UTC with hour, min, sec = 0)
+ *  @param	string			$country_code		Country code
+ *  @param	int			$lastday		Last day is included, 0: no, 1:yes
+ *  @param	int                     $includefriday		Include friday as non working day (-1=use setup, 0=no, 1=yes)
+ *  @param	int			$includesaturday	Include saturday as non working day (-1=use setup, 0=no, 1=yes)
+ *  @param	int			$includesunday		Include sunday as non working day (-1=use setup, 0=no, 1=yes)
+ *  @return   	int|string					Number of non working days or error message string if error
  *  @see num_between_day(), num_open_day()
  */
-function num_public_holiday($timestampStart, $timestampEnd, $country_code = '', $lastday = 0, $includesaturday = -1, $includesunday = -1)
+function num_public_holiday($timestampStart, $timestampEnd, $country_code = '', $lastday = 0, $includefriday = -1, $includesaturday = -1, $includesunday = -1)
```

Comment:
friday is before saturday, however, to keep backward compatibility, when adding a new parameter, it must be added at end of function. Can you move it at end ?